csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -212,14 +215,16 @@ private void TestPreTrainedModelsOpset7And8()
 
             var opsets = new[] { "opset7", "opset8" };
             var modelsDir = GetTestModelsDir();
+            // return if model directories does not exist(build without 'enable_onnx_tests' option)
+            if (!Directory.Exists(modelsDir))
+                return;
```

Comment:
This return statement can lead to silent failures. Anyway to test for **enable_onnx_tests** option directly, and skip tests accordingly? That is safer than  checking if the directory is present (maybe it moved somewhere else accidentally). If there's no easy way to test for **enable_onnx_tests**,  should keep a hard test in C#.